src/lib/api.ts

```diff
@@ -3,7 +3,10 @@ import { removeMarkdown, truncate } from '$lib/string';
 import { marked } from 'marked';
 
 export const ckanUrl = import.meta.env.VITE_CKAN_URL || 'https://data.stadt-zuerich.ch';
+export const backendUrl = import.meta.env.VITE_BACKEND_URL || 'https://ckan-ogdzh.clients.liip.ch';
+export const schemaOrgProfile = import.meta.env.VITE_SCHEMA_ORG_PROFILE || 'stadtzh_schemaorg';
 export const url = (path: string) => `${ckanUrl}/api/3/action/${path}`;
+export const urlBackend = (path: string) => `${backendUrl}/${path}`;
```

Comment:
the naming of those urls is pretty confusing to me. partially was already named that way before and not sure how complicated it would be to refactor it, but `ckanUrl` pointing to the frontend and `backendUrl` pointing to ckan/backend doesn't help reading this 😅  WDYT?